src/app/index.config.coffee

```diff
@@ -69,12 +69,13 @@ angular.module 'mnoEnterpriseAngular'
       }
 
   .config ($httpProvider) ->
-    $httpProvider.interceptors.push ($q, $injector, DEVISE_CONFIG) ->
+    $httpProvider.interceptors.push ($q, $injector, DEVISE_CONFIG, $rootScope) ->
       {
         request: (config) ->
           # Intercept requests made to the API and reset the sessions timeout
           if DEVISE_CONFIG.timeout_in > 0
-            if config.url.includes('mnoe') && !config.url.match(/(sign_out|sign_in|app_instances_sync)/)
+            if config.url.includes('mnoe') && !config.url.match(/(sign_out|sign_in|app_instances_sync)/) && !$rootScope.timeoutSet
```

Comment:
If I understand properly you're only resetting the timeout if it's not set.
I don't think that's correct as subsequent API requests are updating devise `last_active_at` so should also reset the frontend timer.

See:
* [AppInstancesSyncController](https://github.com/maestrano/mno-enterprise/blob/3.3/api/app/controllers/mno_enterprise/jpi/v1/app_instances_sync_controller.rb#L5)
* [MnoEnterprise::ApplicationController](https://github.com/maestrano/mno-enterprise/blob/3.3/core/app/controllers/mno_enterprise/application_controller.rb#L51)

Reply:
Removed the flag as it was indeed preventing requests from resetting the timer every time an API request was received.